bin/bv_packaging

```diff
@@ -876,7 +876,7 @@ def create_debian_packages(options, selected_packages, dependencies):
         os.makedirs(output)
     # Looking for architecture using apt-config if possible
     architecture = re.search(r'APT::Architecture *\"([^\"]*)\";?',
-                             subprocess.Popen(('apt-config', 'dump'), stdout=subprocess.PIPE, stderr=subprocess.STDOUT).communicate()[0])
+                             str(subprocess.Popen(('apt-config', 'dump'), stdout=subprocess.PIPE, stderr=subprocess.STDOUT).communicate()[0],'UTF-8'))
```

Comment:
`str(b'something', 'UTF-8') `only works in python3, not in python2
It can be replaced with `six.ensure_str(b'something')`